semgrep/semgrep/version.py

```diff
@@ -21,8 +21,7 @@
 )
 VERSION_CACHE_PATH = Path(
     os.environ.get(
-        "SEMGREP_VERSION_CACHE_PATH",
-        Path(os.path.expanduser("~")) / ".cache" / "semgrep_version",
+        "SEMGREP_VERSION_CACHE_PATH", Path.home() / ".cache" / "semgrep_version",
```

Comment:
hmm I would be surprised if a tool dropped files in my home folder. I would expect `/tmp`